packages/components/src/components/modal/modal.scss

```diff
@@ -38,7 +38,7 @@
             position: absolute;
             @include mobile {
                 left: 1.6rem;
-                width: calc(100vw - 3.2rem) !important;
+                width: calc(100vw - 4.8rem) !important;
```

Comment:
As discussed, please make the changes from MT5's stylesheet instead of here, as this is the default `width` for most of the `mobile` components.